lec07-pop-models.Rmd

```diff
@@ -1,4 +1,617 @@
 ---
-title: "Qualitative population models in 2 dimensions"
+title: "Population models in two dimensions and numerically solving equations in R"
+author: "Madeleine Bonsma-Fisher"
+output: html_document
 ---
 
+> ### Learning objectives
+>
+> - Review differential equations and one-dimensional phase portraits
+> - Numerically solve differential equations in R using Euler's method and R's ODE solver
+> - Qualitatively analyze two-dimensional models by hand and in R
+
+> ### Lesson outline
+> 
+> Total lesson time: 2 hours
+> 
+> - 
+
+> ### Setup
+> 
+> - `install.packages('deSolve')`
+> - `install.packages('ggplot2')` (or `tidyverse`)
+
+
+## Recap and some tips
+
+   - You can include math in R notebooks using [LaTeX syntax](https://en.wikibooks.org/wiki/LaTeX/Mathematics). 
+
+     `$$\frac{dN}{dt} = r N (1 - N/K)$$` produces 
+     $$\frac{dN}{dt} = r N (1 - N/K)$$
+
+   - **Differential equation**: an equation that describes how a function of variables is related to
+derivatives of those variables. 
+
+   - Drawing **phase portraits** in one dimension:
+
+    ![Logistic equation simulation](https://github.com/UofTCoders/rcourse/raw/master/logistic.gif)
```

Comment:
Are you creating this logistic.gif file? Or is it from the internet? If it is, maybe just link directly to the website.

Reply:
It's been added to the PR now. I made it :)